aikau/src/main/resources/alfresco/renderers/_PublishPayloadMixin.js

```diff
@@ -111,6 +111,42 @@ define(["dojo/_base/declare",
    return declare([ObjectProcessingMixin], {
 
       /**
+       * Whether to publish the topic globally
+       *
+       * @instance
+       * @type {boolean}
+       * @default
```

Comment:
There is no @since here... but in this case I'm not sure we need it, because these have previously been used... so I don't think we need them here.